.github/workflows/ci-tests.yml

```diff
@@ -20,6 +20,11 @@ jobs:
             13
         ]
     steps:
+      - name: Free Disk Space
+        run: |
+          df -h
+          sudo apt-get clean
+          df -h
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v1
```

Comment:
Do you know if `setup-java` cleans up whatever temp space it needs for installing Java?

Reply:
Unclear, but I don't see any deletion code in the action.